src/database/src/output/communication.rs

```diff
@@ -0,0 +1,69 @@
+use serde::Serialize;
+
+use crate::{
+    enums::{CommunicationType, FrequencyUnits},
+    math::Coordinates,
+    sql_structs,
+};
+
+#[serde_with::skip_serializing_none]
+#[derive(Serialize, Debug)]
+/// Represents a communication station at an airport or in an enroute fir
+pub struct Communication {
+    /// The Geographic region where this communication is
+    pub area_code: String,
+    /// The type of communication
+    pub communication_type: CommunicationType,
+    /// The identifier of the airport which this communication is at, if this an airport communication
+    pub airport_ident: Option<String>,
+    /// The identifier of the FIR which this communication is in, if this is an enroute communication
+    pub fir_rdo_ident: Option<String>,
+    /// The frequency of this communication
+    pub frequency: f64,
+    /// The units of the frequency of this communication
+    pub frequency_units: FrequencyUnits,
+    /// The callsign of this communication
+    pub callsign: Option<String>,
+    /// The name of this communication (only defined for enroute communications)
+    pub name: Option<String>,
+    /// The location of this communication
+    pub location: Coordinates,
+}
+
+impl From<sql_structs::AirportCommunication> for Communication {
+    fn from(row: sql_structs::AirportCommunication) -> Self {

```

Comment:
Similar to other comments, does it make sense to bring `sql_structs::AirportCommunication` into scope?

Reply:
It could since it doesent clash with the Communication struct, however for consistency id like to always qualify sql_structs as most of them have very similar names to our output structs and we dont want confusion